library/alloc/src/boxed.rs

```diff
@@ -614,9 +614,48 @@ impl<T, A: Allocator> Box<T, A> {
     /// ```
     /// #![feature(box_into_inner)]
     ///
-    /// let c = Box::new(5);
+    /// // if you have a type `Point` which is `Copy` 
```

Comment:
why the extra `//` in the comments?

Reply:
Are comments not allowed in the example code?

Reply:
> Are comments not allowed in the example code?

It's totally fine.

Reply:
ah misunderstood, should be fine or you could place the comments above the example